desktop/get-started.md

```diff
@@ -3,33 +3,28 @@ description: Docker Dashboard
 keywords: Docker Dashboard, manage, containers, gui, dashboard, images, user manual
 title: Sign in and get started
 redirect_from:
-- /docker-for-windows/
-- /docker-for-windows/index/
-- /docker-for-windows/started/
-- /engine/installation/windows/
-- /installation/windows/
-- /win/
-- /windows/
-- /windows/started/
-- /winkit/
-- /winkit/getting-started/
+- /desktop/linux/
 - /desktop/linux/index/
+- /desktop/mac/
+- /desktop/mac/index/
+- /desktop/windows/
 - /desktop/windows/index/
 - /docker-for-mac/
 - /docker-for-mac/index/
-- /docker-for-mac/mutagen/
-- /docker-for-mac/mutagen-caching/
```

Comment:
This was closely related to `osxfs` / `virtioFS`, so moved it to the settings page (happy to change back though)